### ISSUE TITLE
Module '"ws"' has no exported member 'OpenEvent'

### ISSUE THREAD

**Issue by isipisi89** [Chia-Mine/chia-agent#26]
Can someone tell me what i am missing here?

```
node_modules/chia-agent/daemon/index.d.ts:1:53 - error TS2305: Module '"ws"' has no exported member 'OpenEvent'.
1 import type { CloseEvent, ErrorEvent, MessageEvent, OpenEvent } from "ws";
```

ws has CloseEvent, ErrorEvent, MessageEvent but no OpenEvent .
Am i using the wrong version?

I tired to install "@types/ws": "^8.5.3" but OpenEvent is still missing.

Here is my json:

```
{
  "version": "0.0.1",
  "private": true,
  "name": "xchapi",
  "scripts": {
    "build": "rimraf ./build && tsc",
    "start": "npm run build && node build/index.js",
    "start:dev": "nodemon"
  },
  "dependencies": {
    "axios": "^0.21.1",
    "chia-agent": "^3.0.1",
    "express": "^4.17.1",
    "https": "^1.0.0",
    "mysql": "^2.18.1",
    "request": "^2.88.2",
    "request-promise": "^4.2.6",
    "socket.io": "^4.0.1",
    "sqlstring": "^2.3.2",
    "ssh2shell": "^1.9.4"
  },
  "devDependencies": {
    "@types/express": "4.17.11",
    "@types/node": "14.14.37",
    "@types/websocket": "^1.0.5",
    "@types/ws": "^8.5.3",
    "nodemon": "^2.0.7",
    "rimraf": "^3.0.2",
    "ts-node": "^9.1.1",
    "typescript": "^4.2.3",
    "websocket": "^1.0.34",
    "ws": "^8.5.0"
  }
}
```

**Comment by isipisi89** [comment]
Found it you need to use ws  type [7.4.7](https://www.npmjs.com/package/@types/ws/v/7.4.7) add this mybe to the docs?

**Comment by ChiaMineJP** [comment]
Thank you @isipisi281 for the issue you found.
After I finish reviewing for new 1.3 API, I'll check your suggestion.

**Comment by ChiaMineJP** [comment]
I've read the release note of `ws` module and it says that `OpenEvent` is now removed at `ws@8.0.0`.
https://github.com/websockets/ws/releases/tag/8.0.0
I'll upgrade `ws` used in `chia-agent` to version 8.5.0.

**Comment by ChiaMineJP** [comment]
I've just released v4.0.0.
Please try this one for your issue.
